### DISCUSSION TITLE
GitHub Desktop 3.0.7 Update Problem

### DISCUSSION THREAD

**Discussion by alvarnell** [community/community#31569] [Other Feature Feedback, Questions, & Ideas]
I'm aware of an update to the macOS version of GitHub Desktop, but from within the 3.0.6 app I'm told that I'm up-to-date. I realize it's available from the website, but I always prefer to get automatic notifications.
<img width="453" alt="Screen Shot 2022-08-31 at 9 25 40 PM" src="https://user-images.githubusercontent.com/4943704/187831307-310dbd2e-ebaa-4198-a108-8cb196acdb45.png">

**Comment by steveward** [comment]
@alvarnell apologies for the confusion. For some releases we do staggered rollouts to ensure there aren't any larger issues we aren't aware of. You can hold `Alt` in the `About` dialog to force the app to update.
